src/parser/core/metric.ts

```diff
@@ -1,11 +1,19 @@
-import { AnyEvent } from './Events';
+import { AnyEvent, CombatantInfoEvent } from './Events';
 import Ability from './modules/Ability';
 
 export interface Info {
+  /** The ID of the selected player */
   playerId: number;
+  /** The selected player's spellbook */
   abilities: Ability[];
+  /** The timestamp of the selected fight's start */
   fightStart: number;
+  /** The timestamp of the selected fight's end */
   fightEnd: number;
+  /** The combatantinfo for the selected player */
+  selectedCombatant?: CombatantInfoEvent;
+  /** The combatantinfos for all players in the encounter, indexed by playerId */
+  combatants?: { [playerId: number]: CombatantInfoEvent };
```

Comment:
This should always be set, right?

Reply:
Yes, see above comment (obviously this one's easy to mock because I can just do `[]`)

Reply:
This should be cleaned up in shadowlands now.